app.js

```diff
@@ -1,6 +1,8 @@
 const log = require('./app/lib/logger');
 const app = require('./server');
+const applicationStarts = require('./app/lib/promCounters').applicationStarts;
```

Comment:
Just an observation - the code uses a mix of styles:
```
const applicationStarts = require('./app/lib/promCounters').applicationStarts;
```
vs
```
const counters = require('../lib/promCounters');
[snip]
counters.notFound.inc(1);
```

Reply:
I do and that is why its in. The latter is there due to there being multiple counters used in that file.